docs/editor/settings-sync.md

```diff
@@ -124,30 +124,31 @@ If you want to remove all your data from our servers, just turn off sync via **S
 
 ### Is VS Code Settings Sync the same as the Settings Sync extension?
 
-No, the [Settings Sync](https://marketplace.visualstudio.com/items?itemName=Shan.code-settings-sync) extension by [Shan Khan](https://marketplace.visualstudio.com/publishers/Shan) uses a private Gist on GitHub to share your VS Code settings across different machines and is unrelated to the VS Code Settings Sync.
+No, the [Settings Sync](https://marketplace.visualstudio.com/items?itemName=Shan.code-settings-sync) extension by [Shan Khan](https://marketplace.visualstudio.com/publishers/Shan) uses a secret GitHub Gist to coordinate your private VS Code configuration settings among connected machines, which is unrelated to the VS Code 'Settings Sync' extension.
 
-### What types of accounts can I use for Settings Sync sign in?
+### Which account providers may be used for Settings Sync sign in?
 
-VS Code Settings Sync supports signing in with either a Microsoft account (for example Outlook or Azure accounts) or a GitHub account. Sign in with GitHub Enterprise accounts is not supported. Other authentication providers may be supported in the future and you can review the proposed Authentication Provider API in [issue #88309](https://github.com/microsoft/vscode/issues/88309).
+VS Code Settings Sync currently supports only sign-ins with either Microsoft's (ie, Outlook or Azure accounts) or a non-Enterprise type of GitHub account. Other authentication providers may be supported in the future; please review the proposed Authentication Provider API in [issue #88309](https://github.com/microsoft/vscode/issues/88309).
```

Comment:
The recommendation is not to use Latin abbreviations such as `ie` or `eg`.
https://docs.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/e/eg